data-files/samples/weapons.Experiment.Any

```diff
@@ -0,0 +1,151 @@
+{ 
+    description = "manytargets"; 
+    frameRate = 120; 
+    showHUD = true;
+    renderWeaponStatus = true;
+    cooldownMode = "ring";
+    cooldownColor = Color4(0.7,1.0,0.9,0.75);
+    sessions = ( 
+        { 
+            description = "1 shot"; 
+            id = "1shot"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 20; 
+                    ids = ( "static", "static", "moving"); 
+                } ); 
+            
+            weapon = { 
+                id = "1hit"; 
+                firePeriod = 0.3;
+                damagePerSecond = 4;
+                autoFire = false;
+            }; 
+            
+        }, 
+        { 
+            description = "2 shots"; 
+            id = "2shots"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 20; 
+                    ids = ( "static", "static", "moving"); 
+                } ); 
+            
+            weapon = { 
+                id = "2hit"; 
+                firePeriod = 0.3;
+                damagePerSecond = 2;
```

Comment:
Could add a comment describing how this adds up to a 2-hit kill since the math isn't "exact" (i.e. perfect would be 1.6666...) but could be 1.7 with a comment

Reply:
I'm trying to mostly have these configs "speak for themselves" and leave the comments in the documentation. That said, I don't think we explicitly call out the need to worry about rounding on weapon damage. I'll check.